lib/faq-databases.ts

```diff
@@ -0,0 +1,15 @@
+import database from "../data/wbw-faq-sheets.json";
+
+export type Databases = FAQDATA[];
+
+export type FAQDATA = {
```

Comment:
```suggestion
export type Databases = FaqData[];

export type FaqData = {
```

Please use PascalCase for types so it's easier to split the words, thanks!